lib/src/validator.dart

```diff
@@ -5,6 +5,7 @@
 import 'dart:async';
 
 import 'package:meta/meta.dart';
+import 'package:pub/src/validator/changelog.dart';
```

Comment:
Drive by comment: this import does not follow the pattern of the others. (should be a relative import)